### PR TITLE
add test for dokku_global_cert

### PR DESCRIPTION
**Pull request by ltalirz**
So far, CI is only testing that the role can be installed (with default options).

That's fine, but the role includes a number of features that aren't tested by this approach.
Here, I'm adding a test for #72 as the 'verify' stage (which currently fails, and should be resolved after #73 is merged).

This approach can be used going forward to test any other feature that isn't part of the regular role setup.

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -23,7 +23,3 @@ provisioner:
   name: ansible
   playbooks:
     converge: converge.yml
-scenario:
-  name: default
-verifier:
-  name: testinfra
```
Comment (josegonzalez) [inline molecule/default/molecule.yml:29]: Why this removal?

Comment (ltalirz) [reply molecule/default/molecule.yml:29]: testinfra is no longer the default verifier in molecule 3.0, ansible is (which we're using here)
https://molecule.readthedocs.io/en/latest/configuration.html#verifier

**Comment by ltalirz** [comment]
@josegonzalez If you add a branch protection rule for `master` that branches need to be "up to date", then you'll get a button here to "update" the PR (which should make the tests pass).

**Comment by ltalirz** [comment]
I see you trust my word more than I do ;-)

**Pull request merged**